### PR TITLE
Theming for storybook

### PR DESCRIPTION
**Pull request by bobbonius**
# Description

I've added theming for Storybook.

## How to test

- Checkout this branch
- `$ yarn && yarn storybook`
- Have a look around and confirm all is still working.

## Screenshots

image

### CODE REVIEW

**Comment by glenngijsberts** [comment]
Seems that the prettier test is failing, you can format it with `yarn prettier --write` I think.

File: .storybook/main.js [hunk 1]
```diff
@@ -2,8 +2,11 @@ const path = require('path')
 const toPath = _path => path.join(process.cwd(), _path)
 
 module.exports = {
-  stories: ['../src/**/*.stories.@(ts|js|tsx)'],
-  addons: ['@storybook/addon-essentials'],
+  stories: [
+    '../src/**/*.stories.@(ts|js|tsx)',
+    '../docs/**/*.stories.@(ts|tsx|mdx)',
```
Comment (RobVermeer) [inline .storybook/main.js]: I guess this is part of another story 😉

File: .storybook/preview.js [hunk 2]
```diff
@@ -17,6 +17,11 @@ export const parameters = {
       },
     ],
   },
+  options: {
+    storySort: {
+      order: ['Guidelines', ['Getting Started', 'Design Tokens']],
```
Comment (RobVermeer) [inline .storybook/preview.js]: This is also for another story right?

File: package.json [hunk 3]
```diff
@@ -78,6 +78,7 @@
     "@emotion/jest": "^11.3.0",
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
+    "@storybook/addon-docs": "^6.3.11",
```
Comment (RobVermeer) [inline package.json]: And this as well

File: .storybook/preview.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 import { BaseStyles } from '../src/components/BaseStyles'
 import Theme from '../src/components/Theme'
-import { color } from '../src/theme'
 
 export const parameters = {
+  // layout: 'centered',
```
Comment (RobVermeer) [inline .storybook/preview.js:6]: The things we don't use we can just remove right?

File: .storybook/theme.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+  // UI
+  appBg: 'rgba(0, 182, 240, 0.08)',
+  appContentBg: 'white',
+  appBorderColor: 'white',
+  appBorderRadius: 4,
+
+  // Typography
+  fontBase: '"Open Sans", sans-serif',
+  // fontCode: 'monospace',
```
Comment (RobVermeer) [inline .storybook/theme.js:18]: Same for this one

File: .storybook/theme.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+export default create({
+  base: 'light',
+  brandTitle: 'Solar Design System',
+  brandUrl: 'https://ticketswap.com',
+  brandImage:
+    'https://s3-eu-west-1.amazonaws.com/ticketswap/public/downloads/bm-wm-hor-white-blue.png',
+
+  // UI
+  appBg: 'rgba(0, 182, 240, 0.08)',
```
Comment (RobVermeer) [inline .storybook/theme.js:11]: Would it be an idea to use our own theming colors? So the ones defined at `src/theme.ts`

**Review by RobVermeer — APPROVED** [review]
Looks great, really like it! 😍

File: .storybook/theme.js (hunk 6 above)
Comment (bobbonius) [reply .storybook/theme.js:11]: @RobVermeer Maybe something for a next iteration. As currently, I can't access the css variable from `theme.ts` in `theme.js`. It's in Solar, but not in Storybook. If that makes sense?

**Pull request merged**